pom.xml

```diff
@@ -296,6 +296,12 @@
         <artifactId>powermock-module-testng</artifactId>
         <version>${powermock.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-core</artifactId>
```

Comment:
Why add this dependency?

Reply:
`mockito-core` version 5.6.0 is from `Pulsar` and it requires JDK21. In `Pulsar-Adapters`, it should be downgraded to 3.12.4
`Pulsar` commit: 
<img width="1678" alt="image" src="https://github.com/user-attachments/assets/97220633-956e-42ae-a016-73ab202c501e">